packages/ovee/src/errors/Logger.ts

```diff
@@ -0,0 +1,73 @@
+import { isString } from 'src/utils/isString';
+
+type LogLevel = 'log' | 'warn' | 'info' | 'error';
+
+export class Logger {
+	constructor(public subnamespace = '', public namespace = 'Ovee') {}
+
+	get basePrefix(): string {
+		return this.createPrefix([this.namespace, this.subnamespace]);
+	}
+
+	private createPrefix(parts: string[]) {
+		const prefix = parts.filter(p => !!p).join(' ~ ');
```

Comment:
making the `~` glue configurable through the constructor wouldn't hurt, I guess 🤔

Reply:
sure